.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+<!-- Please complete the following sections as necessary. -->
+
+## Description
+
+<!-- Summary of the changes, related issue, relevant motivation, and context -->
+
+### Ticket
+
+<!-- Link to ticket in pivotal -->
+
+### Approach
+
+<!-- Any changed dependencies, e.g. requires an install/update/migration, etc. -->
+
+### Notes
+
+<!-- Additional information, key learnings, and future development considerations. -->
+
+## Code author checklist
+
+- [ ] I have performed a self-review of my code
+- [ ] My code has adequate test coverage
+- [ ] I have not used any relative imports
+- [ ] I have updated the documentation, if necessary
```

Comment:
I would suggest:

- remove "My code has adequate test coverage" - this is so fundamental that I don't count it as something you need to double-check. It's like having a checkbox for "my code adds behavior that fulfills the Acceptance Criteria"
- "I have updated the documentation, if necessary" - which documentation does this refer to?
- I vote we should add something like "Checked and removed unused config from CMS"

Reply:
So on my previous team we had a literal copy of the AC from the ticket here, just as a recognition that you actually solved all the things involve in the story. But I think thats probably unnecessary at this point.

Reply:
I second Anne's point regarding unused config from CMS and I also think we can also add something regarding removing any functionality/code that is not being used anymore.